tsdb/querier_test.go

```diff
@@ -1400,13 +1400,13 @@ func (m mockIndex) Close() error {
 	return nil
 }
 
-func (m mockIndex) SortedLabelValues(name string) ([]string, error) {
+func (m mockIndex) LabelValues(name string) ([]string, error) {
 	values, _ := m.LabelValues(name)
```

Comment:
You missed this. Can you verify if you have missed anywhere else?

Reply:
My bad, fixed. Strange that the test passed, should have been a stackoverflow you'd think.
Will verify

Reply:
I have verified, they all seemed changed now, assuming that:
All outside calls that were to the previous `LabelValues` to still be to `LabelValues` ?
The return value will be sorted now, which I think is desired, correct me if I'm wrong